frontend/utils/constants.js

```diff
@@ -22,6 +22,8 @@ export const KILOBYTE = 1 << 10
 export const MEGABYTE = 1 << 20
 export const CHAT_ATTACHMENT_SIZE_LIMIT = 30 * MEGABYTE // in byte.
 export const IMAGE_ATTACHMENT_MAX_SIZE = 400 * KILOBYTE // 400KB
+export const CHAT_LONG_MESSAGE_HEIGHT_THRESHOLD_DESKTOP = 750 * 1.25 // in px
+export const CHAT_LONG_MESSAGE_HEIGHT_THRESHOLD_MOBILE = 750 * 1.5 // The value of mobile is more tolerant considering smaller screen size.
```

Comment:
Pls let me know if we would love to change these values. As I said in [this comment](https://github.com/okTurtles/group-income/issues/2728#issuecomment-2760700835) too, the physical height of a message is checked against these thresholds when,

 1. the message is initially rendered
 3. below isPhone state in ChatMain.vue changes:

https://github.com/okTurtles/group-income/blob/a11b226e68e98b1a415ed61a31bb54f544a6686c/frontend/views/containers/chatroom/ChatMain.vue#L250-L252

I haven't used `resize` event here for a performance reason (Running the computation for all messages in the chat every time `resize` event is fired would be really bad. Especially if it's a large chatroom)

Reply:
I feel like both of these values are slightly too long. Could you reduce them from `750` to `500`?